tests/lava/lib/optimization/utils/generators/test_mis.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import unittest
+import networkx as netx
+import numpy as np
+
+from lava.lib.optimization.problems.problems import QUBO
+from lava.lib.optimization.utils.generators.mis import MISProblem, \
+    find_maximum_independent_set, indices_to_binary_vector
+
+
+class TestMISProblem(unittest.TestCase):

```

Comment:
The unit tests are very rough. In the future, they should be extended by tests that verify that output is actually correct (and not just of the correct type).

I understand that we have very limited time right now. What I would do is to go with the current code, and file an issue so that we extend the unit tests asap after the release.

Reply:
Agree on that, I'll go ahead and open the issue. Thanks!